src/Protocol/DFRN.php

```diff
@@ -119,7 +119,9 @@ public static function entries($items, $owner)
 			$item["entry:cid"] = defaults($item, "entry:cid", 0);
 
 			$entry = self::entry($doc, "text", $item, $owner, $item["entry:comment-allow"], $item["entry:cid"]);
-			$root->appendChild($entry);
+			if (isset($entry)) {
```

Comment:
You really dislike ```empty(...)```, do you? :-)

Reply:
To be fair, in this case it kind of makes sense?

Reply:
I'm brainwashed of the Config-refactoring.. There are "empty" strings which are valid :D

Reply:
Since ```empty``` works in most cases, I - personally - would prefer using it whenever possible, and to use ```isset``` only if ```empty``` wouldn't work.